pagodo.py

```diff
@@ -122,6 +122,15 @@ def __init__(
         self.total_urls_found = 0
         self.proxy_rotation_index = 0
 
+        # -o with no filename.  Desire to save results, don't care about the file name.
+        if self.save_pagodo_results_to_json_file is None:
+            self.save_pagodo_results_to_json_file = f"{self.base_file_name}.json"
+
+        # -s with no filename.  Desire to save results, don't care about the file name.
+        if self.save_urls_to_file is None:
+            self.save_urls_to_file = f"{self.base_file_name}.txt"
+
+        import ipdb;ipdb.set_trace()
```

Comment:
This line launches ipdb terminal and script does not continue execution. Specifically:
```
> /home/cgeo/Projects/pagodo/pagodo.py(135)__init__()
    134         # Assign log level.
--> 135         ROOT_LOGGER.setLevel((6 - self.verbosity) * 10)
    136 

ipdb> 
```

Reply:
Good catch